protos/dcs/srs/v0/srs.proto

```diff
@@ -1,17 +1,20 @@
 syntax = "proto3";
-package dcs.tts.v0;
+package dcs.srs.v0;
 import "dcs/common/v0/common.proto";
-option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Tts";
-option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/tts";
+option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Srs";
+option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/srs";
 
-service TtsService {
+service SrsService {
   // Synthesize text to speech and transmit it over SRS. By default, this blocks until a
   // transmission completed (unless `async` is set to `true`). This can be used to prevent
   // transmission to overlap each other, by not sending another transmission on the same frequency
   // until you've received the response from the previous transmission on that frequency. However,
   // it does not block or prevent any other client from transmitting over the same frequency at the
   // same time.
   rpc Transmit(TransmitRequest) returns (TransmitResponse) {}
+
+  // Retrieve a list of units (players) and their active frequencies that are connected to SRS.
+  rpc GetClients(GetClientsRequest) returns (GetClientsResponse) {}
```

Comment:
Bugger, I completely missed this comment. My apologies. I think hacing a separate SRS server is fine and dandy. It may be that we want to add more APIs specific to SRS later on.